### ISSUE TITLE
The library jar picked up from maven repository is having issue.

### ISSUE THREAD

**Issue by mvigneshtuty** [awslabs/amazon-sqs-java-extended-client-lib#18]
The latest source when compiled and build from `https://github.com/awslabs/amazon-sqs-java-extended-client-lib` is **working fine** for deleting messages from SQS backed by S3 for large payloads.
However the same is not working when the jars are picked from maven repository.
Please do resolve it and push the latest one to maven.

**Issue faced when the jar is picked from maven: (when attempting to delete SQS messages)**

```
Exception in thread "main" java.lang.AbstractMethodError: com.amazon.sqs.javamessaging.AmazonSQSExtendedClient.deleteMessage(Lcom/amazonaws/services/sqs/model/DeleteMessageRequest;)Lcom/amazonaws/services/sqs/model/DeleteMessageResult;
    at org.nfjs.aws.sqs.SQSExtendedClient.deleteMessage(SQSExtendedClient.java:149)
    at org.nfjs.aws.sqs.SQSExtendedClient.pollForMessages(SQSExtendedClient.java:139)
    at org.nfjs.aws.sqs.SQSExtendedClient.receiveMessage(SQSExtendedClient.java:108)
    at org.nfjs.aws.sqs.SQSExtendedClient.main(SQSExtendedClient.java:63)
```

**Comment by kuba-aws** [comment]
Please use the 1.0.1 which fixed this behavior and is published to Maven.

**Comment by mvigneshtuty** [comment]
Thank you. I confirm that the issue is addressed in 1.0.1
